features/app-gw.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Test Azure App gateway
+
+    Scenario: Ensure the Azure app gateway has http2 enabled
+        Given I have azurerm_application_gateway defined
+        Then it must contain enable_http2
+        Then its value must be true
+
+    Scenario: Ensure the Azure app gateway is deployed with a basic sku
+        Given I have azurerm_application_gateway defined
+        Then it must contain sku
+        And it must have tier
+        Then its value must be Basic
+
+
+    Scenario: Ensure the Azure app gateway is deployed with an user assigned identity
+        Given I have azurerm_application_gateway defined
+        Then it must contain identity
+        And it must have type
+        Then its value must be UserAssigned
```

Comment:
see the red mark below. it means you need to have a new line at the end of the file. I suppose it's there because although you have a new line, you have also some white space